bonus/specs/bpd/winning_transactions_v2.json

```diff
@@ -0,0 +1,659 @@
+            "description": "Bearer Token",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/WinningTransactionPageResource"
+            }
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "500": {
+            "description": ""
+          }
+        }
+      }
+    },
+    "/total-cashback": {
+      "get": {
+        "description": "getTotalCashback",
+        "operationId": "getTotalScoreUsingGET",
+        "summary": "getTotalCashback",
+        "tags": [
+          "Bonus Pagamenti Digitali winning-transaction Controller"
+        ],
+        "parameters": [
+          {
+            "name": "awardPeriodId",
+            "in": "query",
+            "description": "awardPeriodId",
+            "required": true,
+            "type": "integer"
+          },
+          {
+            "name": "Authorization",
+            "in": "header",
+            "description": "Bearer Token",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/TotalCashbackResource"
+            }
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "404": {
+            "description": "Not Found"
+          },
+          "500": {
+            "description": ""
+          }
+        }
+      }
+    },
+    "/countbyday": {
+      "get": {
+        "description": "getCountByDay",
+        "operationId": "603564be9dbc61ba660487f6",
+        "summary": "getCountByDay",
+        "parameters": [
+          {
+            "name": "hpan",
+            "in": "query",
+            "description": "id dello strumento di pagamento, che corrisponde all'hash del PAN (Primary Account Number) del metodo di pagamento",
+            "type": "string"
+          },
+          {
+            "name": "awardPeriodId",
+            "in": "query",
+            "description": "awardPeriodId",
+            "required": true,
+            "type": "integer"
+          },
+          {
+            "name": "Authorization",
+            "in": "header",
+            "description": "Bearer Token",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/TrxCountByDayResourceArray"
+            }
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "500": {
+            "description": ""
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "BpdWinning-transactionsGet200ApplicationJson;charset=UTF-8Response": {
```

Comment:
This will cause an error on api model generation with io-utils
Maybe this definition is unnecessary since it is never used by any API spec